File: SDKs/motaword.com/1.0/typescript/src/sdk/models/shared/languagepair.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class LanguagePair extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=source_language" })
  sourceLanguage?: string;

  @SpeakeasyMetadata({ data: "json, name=target_language" })
  targetLanguage?: string;
}

File: SDKs/motaword.com/1.0/python/src/sdk/models/shared/languagepair.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class LanguagePair:
    source_language: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('source_language') }})
    target_language: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('target_language') }})
    
